Mathlib/Order/SuccPred/Basic.lean

```diff
@@ -1443,6 +1443,26 @@ variable [LinearOrder α]
 
 section SuccOrder
 
+theorem succ_min [SuccOrder α] (x y : α) :
+    succ (min x y) = min (succ x) (succ y) := by
+  cases le_total x y with
+  | inl h =>
+    rw [min_eq_left h, min_eq_left]
+    simp [h]
+  | inr h =>
+    rw [min_eq_right h, min_eq_right]
+    simp [h]
+
+theorem succ_max [SuccOrder α] (x y : α) :
+    succ (max x y) = max (succ x) (succ y) := by
```

Comment:
@YaelDillies, is this true as
```
theorem succ_min [SemilatticeSup α] [SuccOrder α] (x y : α) :
    succ (x ⊔ y) = succ x ⊔ succ y := by
```

Reply:
No

Consider, three copies of `Nat`, two are incomparable and the third one is greater than both. Then `a in bottom left copy ⊔ b in bottom right copy = 0 in top copy`, so
* `succ (0 in bottom left copy) ⊔ succ (0 in bottom right copy) = 1 in bottom left copy ⊔ 1 in bottom right copy = 0 in top copy`
* `succ (0 in bottom left copy ⊔ 0 in bottom right copy) = succ (0 in top copy) = 1 in top copy`.